docs/v1/P4Runtime-Spec.mdk

```diff
@@ -511,6 +511,15 @@ In this situation, P4Info is selectively packaged into role-based subsets to
 allow some controllers access to just the functionality required. For example, a
 controller may only need read access to statistics counters and nothing more.
 
+## P4Runtime State Across Restarts { #sec-restarts }
+
+All targets support full restarts, where all forwarding state is reset and the
+P4Runtime server starts with a clean state.  Some targets may also support
+In-Service Software Upgrade (ISSU), where the software on the target can be
+restarted while traffic is being forwarded. In this case, the P4Runtime Server
```

Comment:
This may sounds a little too restrictive. Perhaps something like: "In this case, the P4Runtime server may have the ability to access information from memory before the upgrade".

Reply:
Done.